initials.c

```diff
@@ -0,0 +1,12 @@
+#include <stdio.h>
```

Comment:
Overall, great job! You did forget the return 0; which you should include as good practice. Other than that, your code runs and you provided comments!